eu-west-1/cluster.json

```diff
@@ -130,7 +130,7 @@
           "Fn::Join": [
             "",
             [
-              "IyEvYmluL2Jhc2ggLXYNCiMgTWFkY29yZSBLdWJlcm5ldGVzIE5vZGUgUHJlIENvbmZpZ3VyZQ0KIyBNYWludGFpbmVkIGJ5IE1hZGNvcmUgTHRkIChodW1hbnNAbWFkY29yZS5haSkNCg0Kc3VkbyBlY2hvIEVOVj1BV1MgPj4gL2V0Yy9lbnZpcm9ubWVudA0KDQpLVUJfTUFTVEVSX0lQPSIke0tVQl9NQVNURVJfSVA6LTEwLjk5LjEwMS45OX0iDQoNCiMgUFJFQ09ORklHVVJFIENPTlRST0xCT1gNCnN1ZG8gYXB0LWdldCB1cGRhdGUNCnN1ZG8gYXB0LWdldCBpbnN0YWxsIGdpdCAteQ0Kc3VkbyBta2RpciAtcCAvb3B0L2NvbnRyb2xib3gNCnN1ZG8gY2hvd24gdWJ1bnR1OnVidW50dSAvb3B0L2NvbnRyb2xib3gNCmdpdCBjbG9uZSBodHRwczovL2JpdGJ1Y2tldC5vcmcvcm9uYWFuaW1hdGlvbi9jb250cm9sYm94LmdpdCAvb3B0L2NvbnRyb2xib3gNCg0Kc3VkbyBjaG1vZCAreCAvb3B0L2NvbnRyb2xib3gva3ViZXJuZXRlcy9jbHVzdGVyL2Nmbi1pbml0LnNoDQpzdWRvICIvb3B0L2NvbnRyb2xib3gva3ViZXJuZXRlcy9jbHVzdGVyL2Nmbi1pbml0LnNoIg0KDQpzdWRvIGNobW9kICt4IC9vcHQvY29udHJvbGJveC9rdWJlcm5ldGVzL2NsdXN0ZXIvbm9kZS1pbnN0YWxsLnNoDQpzdWRvICIvb3B0L2NvbnRyb2xib3gva3ViZXJuZXRlcy9jbHVzdGVyL25vZGUtaW5zdGFsbC5zaCINCg"
+              "IyEvYmluL2Jhc2ggLXYNCiMgTWFkY29yZSBLdWJlcm5ldGVzIE5vZGUgUHJlIENvbmZpZ3VyZQ0KIyBNYWludGFpbmVkIGJ5IE1hZGNvcmUgTHRkIChodW1hbnNAbWFkY29yZS5haSkNCg0Kc3VkbyBlY2hvIEVOVj1BV1MgPj4gL2V0Yy9lbnZpcm9ubWVudA0KDQpLVUJfTUFTVEVSX0lQPSIke0tVQl9NQVNURVJfSVA6LTEwLjk5LjEwMS45OX0iDQoNCiMgUFJFQ09ORklHVVJFIENPTlRST0xCT1gNCnN1ZG8gYXB0LWdldCB1cGRhdGUNCnN1ZG8gYXB0LWdldCBpbnN0YWxsIGdpdCAteQ0Kc3VkbyBta2RpciAtcCAvb3B0L2NvbnRyb2xib3gNCnN1ZG8gY2hvd24gdWJ1bnR1OnVidW50dSAvb3B0L2NvbnRyb2xib3gNCmdpdCBjbG9uZSBodHRwczovL2JpdGJ1Y2tldC5vcmcvcm9uYWFuaW1hdGlvbi9jb250cm9sYm94LmdpdCAvb3B0L2NvbnRyb2xib3gNCg0Kc3VkbyBjaG1vZCAreCAvb3B0L2NvbnRyb2xib3gva3ViZXJuZXRlcy9jbHVzdGVyL2Nmbi1pbml0LnNoDQpzdWRvICIvb3B0L2NvbnRyb2xib3gva3ViZXJuZXRlcy9jbHVzdGVyL2Nmbi1pbml0LnNoIg0KDQpzdWRvIGNobW9kICt4IC9vcHQvY29udHJvbGJveC9rdWJlcm5ldGVzL2NsdXN0ZXIvbm9kZS1pbnN0YWxsLnNoDQpzdWRvICIvb3B0L2NvbnRyb2xib3gva3ViZXJuZXRlcy9jbHVzdGVyL25vZGUtaW5zdGFsbC5zaCINCg=="
```

Comment:
So, after I use this I need to get each piece of string and make base64 encode, separately ?